example/flake.nix

```
@@ -76,6 +76,25 @@
               };
             };
           };
+
+        # nix run .#ponysay up to start the process
+        packages.ponysay = (import ../nix/eval-modules.nix).makeProcessCompose {
```

This would look something like:

```suggestion
        packages.ponysay = inputs.process-compose-flake.lib.makeProcessCompose {
```

---

Doesn't this mean that the example can only be run *after* pushing the code upstream? I think it's nice if it's possible to use the example while working on the code. But I can surely do it.

---

I'll update the lock file after merging the PR, but in CI, it will always use latest, due to https://github.com/Platonic-Systems/process-compose-flake/blob/f6ce9481df9aec739e4e06b67492401a5bb4f0b1/flake.nix#L15

---

In the meanwhile, you can do `cd ./example && nix run . --override-input process-compose-flake ..` to test locally.

---

Done.